src/vulkan.c

```diff
@@ -317,6 +317,14 @@ GLFWAPI VkResult glfwCreateWindowSurface(VkInstance instance,
         return VK_ERROR_EXTENSION_NOT_PRESENT;
     }
 
+    if (window->context.client != GLFW_NO_API)
+    {
+        _glfwInputError(GLFW_INVALID_VALUE,
+                        "Vulkan: Window surface creation requires the window "
+                        "to have the client API set to GLFW_NO_API");
```

Comment:
Error string literals should be kept whole even past 80 to help people who grep for them.

Reply:
Done